sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/UniversalControlFlowGraphBuilder.cs

```diff
@@ -334,6 +370,40 @@ private Instruction CreateInstruction(SyntaxNode syntaxNode, string methodId, st
 
             private ISymbol GetSymbol(SyntaxNode syntaxNode) =>
                 semanticModel.GetSymbolInfo(syntaxNode).Symbol;
+
+            public void CreateAttributeInstructions(IParameterSymbol parameter)
+            {
+                foreach (var attribute in parameter.GetAttributes().Where(a => a.AttributeConstructor != null))
```

Comment:
With this code you are only handling attributes specifically set on parameters, forgetting the ones set at method level, is that right? Is that what is expected?

Reply:
I think that's a more general question that needs to be answered by Marcelo and Dinesh.

Reply:
The goal of SONARSEC-27 is to provide minimal support for ASP.NET Core MVC. It mentions `[Controller]` and `[Validation]`.

As part of this ticket, I expect:
* `[Controller]` to produce `__entrypoint` (see SONARSEC-41)
* `[Validation]` attributes (on parameters) to produce `__annotation`